content/docs/about/community.md

```diff
@@ -43,24 +43,38 @@ mailing list will automatically send you calendar invitations for the meetings,
 Agenda, notes, and a reminder of the next call are sent to the kubeflow-discuss
 mailing list.
 
-### Forums and mailing groups
-
-Summary:
-
-* [Invitation to our Slack channel](https://join.slack.com/t/kubeflow/shared_invite/enQtNDg5MTM4NTQyNjczLTdkNTVhMjg1ZTExOWI0N2QyYTQ2MTIzNTJjMWRiOTFjOGRlZWEzODc1NzMwNTMwM2EzNjY1MTFhODczNjk4MTk)
-* [Twitter](http://twitter.com/kubeflow)
-* [Mailing list: kubeflow-discuss](https://groups.google.com/forum/#!forum/kubeflow-discuss)
-
-<br>
-More detail:
-
-| Topic                                                           | Mailing list                                                                      | Slack                                                                                             |
-| :----                                                           | :------------                                                                     | :-------------                                                                                    |
-| General discussion                                              | [kubeflow-discuss](https://groups.google.com/forum/#!forum/kubeflow-discuss)      | [#general](https://kubeflow.slack.com/messages/C7REE0EHK)                                         |
-| TF Operator ([GitHub](https://github.com/kubeflow/tf-operator)) | [tf-operator](https://groups.google.com/a/kubeflow.org/forum/#!forum/tf-operator) | [#tf-operator](https://kubeflow.slack.com/messages/C985VJN9F) |
-| Community meeting chat                                          | n/a                                                                               | [#community](https://kubeflow.slack.com/messages/C8Q0QJYNB)                                       |
-
-Slack server: [kubeflow.slack.com](https://kubeflow.slack.com/)
+<a id="slack"></a>
+### Slack online chat and messaging
```

Comment:
I'd call this "Slack Community & Channels" instead.

Reply:
Done! PTAL.